specification/cognitiveservices/ContentSafety/routes.tsp

```diff
@@ -72,15 +157,48 @@ interface ImageOperations {
   @post
   analyzeImage is Azure.Core.RpcOperation<
     {
-      @doc("The image analysis request.")
       @bodyRoot
+      @doc("The image analysis request.")
       @clientName("options", "csharp")
       @clientName("options", "python")
       @clientName("options", "java")
       body: AnalyzeImageOptions;
     },
     AnalyzeImageResult
   >;
+
+  @added(ContentSafety.Versions.v2024_09_15_Preview)
+  @summary("Analyze Image with Image Incidents")
+  @doc("A synchronous API to analyze image content with the custom content safety Image Incidents defined.")
+  @route("/image:detectIncidents")
+  @post
+  detectImageIncidents is Azure.Core.RpcOperation<
+    {
+      @bodyRoot
+      @doc("The request body to be detected.")
+      @clientName("options", "csharp")
+      @clientName("options", "python")
+      @clientName("options", "java")
+      body: AnalyzeImageDetectIncidentsOptions;
+    },
+    DetectIncidentsResult
+  >;
+}
+
+@added(ContentSafety.Versions.v2024_09_15_Preview)
+interface ImageWithTextOperations {
+  @summary("Analyze Image with Text")
+  @doc("A synchronous API to analyze image content with embedded or custom text content.")
+  @route("/imageWithText:analyze")
+  @post
+  analyzeImageWithText is Azure.Core.RpcOperation<
+    {
+      @doc("The request body of image-with-text analysis.")
+      @body
```

Comment:
Why do you wrap this body with `at-body`?

Reply:
We just follow the previous implementation. Could you please advise what's the best practice here?